### PR TITLE
feat(gitlab-ci): Add openshift platform to mdap ci for v1.0.x branch

### PR DESCRIPTION
**Pull request by nsathyaseelan**
Signed-off-by: nsathyaseelan <sathyaseelan.n@mayadata.io>

- Enable e2e pipelines to run on openshift for v1.0.x branch

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -56,18 +56,16 @@ baseline-image:
...
      - cd e2e-infrastructure/baseline
      - ansible-playbook commit-writer.yml --extra-vars "branch=$BRANCH repo=$CI_PROJECT_NAME commit=$COMMIT"
      - git status
      - git add baseline
      - git status
      - git commit -m "updated $CI_PROJECT_NAME commit:$COMMIT"
-     - git push  https://$user:$pass@github.com/openebs/e2e-infrastructure.git --all
-     - curl -X POST -F variable[INFRA_BRANCH]=$BRANCH -F token=$PACKET -F ref=k8s-1-11 https://gitlab.openebs.ci/api/v4/projects/27/trigger/pipeline
-     - curl -X POST -F variable[INFRA_BRANCH]=$BRANCH -F token=$PACKET -F ref=k8s-1-12 https://gitlab.openebs.ci/api/v4/projects/27/trigger/pipeline
-     - curl -X POST -F variable[INFRA_BRANCH]=$BRANCH -F token=$PACKET -F ref=k8s-1-13 https://gitlab.openebs.ci/api/v4/projects/27/trigger/pipeline
```
Comment (prateekpandey14) [inline .gitlab-ci.yml]: Are we not using packer cluster now @nsathyaseelan ?

Comment (nsathyaseelan) [reply .gitlab-ci.yml]: we are using packet cluster for master branch commits.

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -56,18 +56,16 @@ baseline-image:
      - echo $BRANCH
      - export COMMIT=${CI_COMMIT_SHORT_SHA}
      - echo $COMMIT
-     - git clone https://github.com/openebs/e2e-infrastructure.git
+     - git clone https://github.com/$REPO/e2e-infrastructure.git
```
Comment (atulabhi) [inline .gitlab-ci.yml]: @nsathyaseelan why we changed from openebs to $REPO ?
 Is it passed as environment variable in gitlab?

File: .gitlab-ci.yml [hunk 3]
```diff
@@ -76,7 +74,7 @@ clean-maya:
     refs:
       - /^(v[0-9][.][0-9][.]x|master)?$/
   script:
-    - sudo minikube delete || true
+    - sudo minikube delete
```
Comment (atulabhi) [inline .gitlab-ci.yml]: removing || true can cause build failure in some cases need to handle it @nsathyaseelan

File: .gitlab-ci.yml (hunk 1 above)
Comment (nsathyaseelan) [reply .gitlab-ci.yml]: @prateekpandey14  are not triggering pipelines in a packet for v1.o.x branches. we are using packet for the master branch.

**Pull request merged**